backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -381,7 +379,7 @@ type TypeReference =
 type Expr =
   | EInt of id * int64
   | EBool of id * bool
-  | EString of id * List<StringSegment>
+  | EString of id * NEList<StringSegment>
```

Comment:
We could represent `""` as either `EString([])` or `EString([""])`. I'm ok with either.

Reply:
I have a slight leaning towards `EString([])` being OK, and not doing NEList here